src/components/scheduling/goals/SchedulingGoalAnalysesBadge.svelte

```diff
@@ -28,19 +29,34 @@
 </script>
 
 {#if currentAnalysis}
-  <div class="scheduling-goal-analyses-badge" class:disabled={!enabled}>
+  <div class="scheduling-goal-analyses-badge st-typography-bold" class:disabled={!enabled}>
     {#if satisfyingActivitiesDifference !== null}
-      <div class="difference-badge" class:disabled={!enabled}>
+      <div
+        class="difference-badge"
+        class:disabled={!enabled}
+        use:tooltip={{ content: 'New Satisfied Activities', maxWidth: 'none', placement: 'top' }}
+      >
         +{satisfyingActivitiesDifference}
       </div>
     {/if}
-    {currentAnalysis.satisfying_activities.length}
+    <!-- TODO what do these things actually mean? -->
```

Comment:
What do you mean here?

Reply:
Ah I think I meant to check in with folks - I wasn't 100% clear on what these different numbers meant in the display. Was writing tooltips for them so wanted to make sure I had the right idea.

Reply:
Got it, I think they are good!